### ISSUE TITLE
Docker compose does not parse environment variables consistently

### ISSUE THREAD

**Issue by KuipersT** [buggyrace/buggy-race-server#73]
We are using Docker Compose for building and running the containers however there are two ways of running that:

1. via docker-compose
2. via docker compose (after installing the the Docker compose CLI plugin

When running with (1), it complains that the DATABASE_URL* is malformed. However, (2) runs just fine. In addition, with (1) Flask crashes and asks for a SECRET_KEY var to be set, but again this cannot be replicated via (2).

This is rather annoying because the use of these two options are not standardised installing (2) which is most ideal is a further step that many docker instances won't be set up for. We should ideally be able to provide the container and not have to get additional software installed on any host OS.

Bottom line is that SECRET_KEY and the complete database URL can be written out manually. This potentially introduces the possibility for syntax errors though if someone removes a colon when editing the password for example. The resultant errors generated by the app will not likely be something that the end-user will investigate and thus pose the risk of "huh, doesn't work out of the box, so bin" attitude.

* DATABASE_URL=$DB_DRIVER://$DB_USER:$DB_PASS@$DB_HOST:$DB_PORT/$DB_NAME
